### DISCUSSION TITLE
Choosing a specific skin for a vehicle in a faction?

### DISCUSSION THREAD

**Discussion by FightKnight441** [KillahPotatoes/KP-Liberation#960] [Q&A]
Hi guys, I am creating a Faction, and want to use the base Arma KA-60 Helicopter in that faction. However, I wish to use the black skin on it, as the CSAT one will not align with the faction's camo. Is there any way to specify the skin to spawn the vehicle with? 

Thanks!

**Comment by KiwiTwitches** [comment]
Reccomend looking into the Arma object configs on their wiki. I believe there's a black variant under the protected units.

As for custom skins, Not certain, but there may be a way to make the unit it calls also apply a skin, but I can't confirm that as I haven't tried
